### DISCUSSION TITLE
Is it possible to pull with ours/theirs flags?

### DISCUSSION THREAD

**Discussion by anareaty** [Vinzent03/obsidian-git#514] [Q&A]
I usually use these flags to avoid conflicts. Is trere any way to use them with this plugin?

**Comment by Vinzent03** [comment]
Please explain what you are talking about. I don't know these flags.

**Comment by anareaty** [comment]
I'm talking about merge strategies, explained here: https://git-scm.com/docs/git-pull

> **ours**
> This option forces conflicting hunks to be auto-resolved cleanly by favoring our version. Changes from the other tree that do not conflict with our side are reflected in the merge result. For a binary file, the entire contents are taken from our side.


>**theirs**
>This is the opposite of ours; note that, unlike ours, there is no theirs merge strategy to confuse this merge option with.

For example, I can pull with this command:

```
git pull -s recursive -X theirs
```

Then, in case of the conflicts remote version of the files would be automatically chosen, so I don't have to resolve conflict manually.
